gax-java/gax/src/main/java/com/google/api/gax/longrunning/OperationTimedPollAlgorithm.java

```diff
@@ -77,6 +87,13 @@ public boolean shouldRetry(TimedAttemptSettings nextAttemptSettings)
     if (super.shouldRetry(nextAttemptSettings)) {
       return true;
     }
+    if (LOGGER.isLoggable(Level.SEVERE)) {
```

Comment:
QQ, is there a reason `SEVERE` logging level was picked? It seems like that is meant for more fatal application issues (and less for an RPC timeout), but I also am not sure if there is a better level (Info? Warning?).

Reply:
That makes sense. My reasoning was more like "exception=severe", but in the context of a timeout on a request I agree something like `WARNING` is better

Reply:
I decreased it to WARNING